### ISSUE TITLE
Maybe WebExtension for SSleuth (wiki)

### ISSUE THREAD

**Issue by rugk** [ghacksuserjs/ghacks-user.js#218]
Sad that your wiki is not editable. Do you want to change this?

In any case please edit SSLeuth on https://github.com/ghacksuserjs/ghacks-user.js/wiki/Appendix-B:-Firefox-Add-ons and mark it with the :purple_heart:. The author says in https://github.com/sibiantony/ssleuth/issues/78#issuecomment-320638840:
> If Mozilla does implement this soon enough, I can offer a web extension version to use with Firefox 57+. ( With the minimal functionality without configuration, domains watching etc. all of which can be added eventually. )

**Comment by earthlng** [comment]
> Sad that your wiki is not editable. Do you want to change this?

atm no

> In any case please edit SSLeuth 

done, thank you

**Comment by Thorin-Oakenpants** [comment]
That's good news. I like SSleuth. I hope it happens.

I can't provide any links, but I have seen enough in comments throughout bugzilla and elsewhere to **know** that a lot of WE APIs will be coming to bring back some lost functionality - they just had to get on with other things first. The dust will settle, and people will be available to start working on these things.

I saw a mozilla dev (I think he is, the name rings a bell) the other day developing a DL manager thingy - essentially replicating down them all AFAIC - here it is: https://www.reddit.com/r/firefox/comments/6uqpi4/download_manager_what_features_would_you_like_to/  .. anyway, totally awesome (for those who like it), listens, knows his stuff.

I think a lot of Moz devs will push for changes too. Some nudging from the likes of EFF, *significant* devs like gorhill et al also get listened to .. it will get there. Feeling so much better than I did 6 months ago when so much was in the dark
